src/index.js

```diff
@@ -460,6 +461,33 @@ export function removeContext(sequelize) {
   shimmer.unwrap(sequelize.constructor.Association.BelongsToMany.prototype, 'get');
 }
 
+export function clearCache(context, sequelize) {
+  const {loaders} = context;
+
+  for (const Model of Object.values(sequelize.models)) {
+    values(Model.associations).forEach(association => {
+      if (association.associationType === 'BelongsTo') {
+        const Target = association.target;
+        if (association.targetKey !== Target.primaryKeyAttribute) {
+          loaders[Target.name].bySingleAttribute[association.targetKey].clearAll();
+        }
+      } else if (association.associationType === 'HasOne') {
+        const Target = association.target;
+        loaders[Target.name].bySingleAttribute[association.foreignKey].clearAll();
+      }
+    });
+    Object.keys(loaders[Model.name]).forEach(key => {
+      // allow removing custom dataloaders, added outside of dataloader-sequelize
```

Comment:
Not sure this is prudent, we probably shouldn't attempt to reason about what the user might want to have happen outside of the context of this library.

Reply:
Hmm actually, since they are added to the loaders object created by the library, it's probably fine. Are you adding custom loaders to the same object in your codebase?

Reply:
Yes, I share your concern and yes I am.

Reply:
Ok, I will add some tests